Revamp documentation to better describe current AoD status (#377)
EGI-Federation/documentation: content/en/users/getting-started/_index.md

     Agreement (SLA) with you
   - A new VO will be created for your community

## Pilot your application

> EGI offers a plaground allocation for users get access to the services and
> understand how to port applications and develop new data analytics tools that
> can be turn into online services that can be accessed by scientist worldwide.

Reviewer: ```suggestion
EGI offers a playground allocation for users to get access to the services and
understand how to port applications and develop new data analytics tools that
can be turn into online services that can be accessed by scientist worldwide.
```

### Requirements and user registration

Access **requires** acceptance of
[Acceptable Use Policy (AUP) and Conditions of the \'EGI Applications on Demand Service\'](https://documents.egi.eu/public/ShowDocument?docid=2635).

{{% alert title="Acknowledgment" color="info" %}}

Users of the service are asked to provide appropriate acknowledgement of the use
in scientific publications. The following acknowledgement text can be used for

> this purpose (you can adapt to match the exact providers in your case):

Reviewer: ```suggestion
this purpose (you should adapt to match the exact providers in your case):
```

**This work used advanced computing resources from the 100%IT, CESGA, CLOUDIFIN,
CYFRONET-CLOUD, GSI-LCG2, IFCA-LCG2, IN2P3-IRES, INFN-CATANIA-STACK,
INFN-PADOVA-STACK, SCAI, TR-FC1-ULAKBIM, UA-BITP and UNIV-LILLE resource centres
of the EGI federation. The services are co-funded by the EGI-ACE project (grant
number 101017567).** {{% /alert %}}

When requesting access users are guided through a registration process. Members
of the EGI support team will perform a lightweight vetting process to validate

> the users\' requests before granting the access to the resources.

Reviewer: ```suggestion
the users' requests before granting the access to the resources.
```
Should not be needed, right?

### Get access to pilot allocation

> 1. Login to the [EGI Marketplace](https://marketplace.egi.eu) with the

Reviewer: ```suggestion
1. Log into the [EGI Marketplace](https://marketplace.egi.eu) with the
```
